### ISSUE TITLE
Programmatic Sermon Series' 

### ISSUE THREAD

**Issue by ksigWyatt** [ThriveCommunityChurch/ThriveChurchOfficialApp#95]
## User Story
As a developer I want to be able to display sermon series' to a user, so that they can see them.

### Acceptance Criteria
- [x] JSON file is available to the user so we can serve them their request
- [x] The file should contain a list of all the sermon series' in a list by year in chronological order
- [x] The link to the page should either be to the website or to the list of URL links
- [x] Will need to be able to be updated frequently and easily outside of application updates. So the JSON cannot reside on the Client device.

## Tasks
- [x] UICollectionVew 
- [x] Make API requests to the API we're making to retrieve the data
  - Will need to figure out how to debug via IIS on LAN from the Mac since Extensions.Primitives v2.1.6 is a "Downgrade" and we need our dependencies to all work before we can debug on the Mac when cloning to the mac
- [x] Create a custom / Flow layout for our collection view
- [x] Make the sort order of the sermons the start date in Descending order.  

## Stretch Goals
- [x] Live Media player - we will have to include a LiveFeed string field at the beginning so we can see if there is currently a live feed.

**Comment by ksigWyatt** [comment]
This story is largely blocked by the API stories from our Mobile_API repo. Once the SermonsController is developed this story can be fully finished. I'd like to use that API to update this JSON within this story.
